### PR TITLE
LPS-67821 Use defined entry points

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

**Comment by carloslancha** [comment]
@jbalsas delete commit "Remove unnecesary configuartion"

**Comment by jbalsas** [comment]
ci:reopen

**Comment by jbalsas** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+import Soy from 'metal-soy';
+import { core } from 'metal';
+
+import 'metal-dropdown';
+
+import CompatibilityEventProxy from 'frontend-js-web/liferay/CompatibilityEventProxy.es'
+
+import templates from './TranslationManager.soy';
+
+class TranslationManager extends Component {
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js]: Docs

File: modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+		this.availableLocales = this.availableLocales;
+
+		this.editingLocale = localeId;
+	}
+
+	/**
+	 * Registers another EventTarget as a bubble target.
+	 *
+	 * @param  {!Object} target
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js]: Docs

File: modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+		this.startCompatibility_();
+	}
+
+	/**
+	 * Add a language to the available locales list and set it as the
+	 * current editing language.
+	 *
+	 * @param  {MouseEvent} event
+	 */
+	addLocale(event) {
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js]: lint, unneeded?

File: modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+	addTarget(target) {
+		this.compatibilityEventProxy_.addTarget(target);
+	}
+
+	/**
+	 * Change the default language.
+	 *
+	 * @param  {MouseEvent} event
+	 */
+	changeDefaultLocale(event) {
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js]: unneeded?

File: modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+		this.resetEditingLocale_();
+
+		this.startCompatibility_();
+	}
+
+	/**
+	 * Add a language to the available locales list and set it as the
+	 * current editing language.
+	 *
+	 * @param  {MouseEvent} event
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js]: Docs

File: modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+			{
+				host: this,
+				namespace: 'translationmanager'
+			}
+		);
+	}
+	
+}
+
+TranslationManager.STATE = {
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.es.js]: docs

**Pull request closed without merging**